### ISSUE TITLE
Can you share the comparison method code?

### ISSUE THREAD

**Issue by kongbo96** [muhanzhang/IGMC#15]
Such as the implement of PinSage.

**Comment by muhanzhang** [comment]
Hi! We decided not to share the implementations of the baselines, as it is beyond our capability to maintain them and answer the related questions. But I can share how we implement PinSage: we directly modified the GC-MC tensorflow code, by replacing its one-hot features with node content features (if any) or node degrees. We didn't adopt PinSage's original random-walk based sampling method, as our graphs are not large; all nodes use GCN to aggregate their neighbors directly. We also did some modifications to adapt PinSage to rating prediction.
